kalite/settings/__init__.py

```diff
@@ -46,6 +46,8 @@ def package_selected(package_name):
         RemovedInKALite_v015_Warning
     )
 
+DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP = getattr(local_settings, "DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP", False)
```

Comment:
Okay, if you want to retrieve it from local_settings, I can see the point... but are you still using local_settings for OSX?